Testing/VelaptorTests/OpenGL/GLWindowTests.cs

```diff
@@ -825,13 +1166,59 @@ public void GLWindow_WhenWindowResizes_SetsGLViewportAndTriggersResizeEvent()
             window.Show();
 
             // Act
-            this.mockWindowFacade.Raise(w => w.Resize += null, null, new WindowSizeEventArgs(30, 40));
+            this.mockSilkWindow.Raise(e => e.Resize += It.IsAny<Action<Vector2D<int>>>(), new Vector2D<int>(11, 22));
+
+            // Assert
+            this.mockGL.Verify(m => m.Viewport(0, 0, 11, 22));
+            this.mockRenderer.Verify(m => m.OnResize(new SizeU(11u, 22u)), Times.Once);
+            Assert.Equal(11u, actualSize.Width);
+            Assert.Equal(22u, actualSize.Height);
+        }
+
+        [Fact]
+        public void GLWindow_WhenUpdatingWhileShuttingDown_DoesNotUpdateAnything()
+        {
+            // Arrange
+            var windowUpdateInvoked = false;
+            var window = CreateWindow();
+            window.Show();
+            window.Update = _ => windowUpdateInvoked = true;
+
+            // Act
+            this.mockSilkWindow.Raise(e => e.Closing += null);
+            this.mockSilkWindow.Raise(e => e.Update += It.IsAny<Action<double>>(), 0.016);
+
+            // Assert
+            Assert.False(windowUpdateInvoked, $"{nameof(GLWindow.Update)} should not of been invoked during window shutdown.");
+            this.mockMouseInput.VerifyNever(m => m.SetScrollWheelSpeed(It.IsAny<int>()));
+            this.mockMouseInput.VerifyNever(m => m.SetScrollWheelDirection(It.IsAny<MouseScrollDirection>()));
+        }
+
+        [Fact]
+        public void GLWindow_WhenUpdatingWhenNotShuttingDown_PerformsUpdate()
```

Comment:
Refactor unit test name
```suggestion
        public void GLWindow_WhileUpdatingWhenNotShuttingDown_PerformsUpdate()
```